hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -389,6 +397,10 @@ public boolean isAsyncClean() {
     return Boolean.parseBoolean(props.getProperty(HoodieCompactionConfig.ASYNC_CLEAN_PROP));
   }
 
+  public boolean isInlineCleaning() {
```

Comment:
this does not read easily. rename `inlineCleaningEnabled` or `shouldCleanInline()` or something like tht? (same wherever applicable)

Reply:
I kept the same naming convention as before for `isInlineCompaction` etc. I refactored all of them to `inline<..>Enabled`